### ISSUE TITLE
No extension problems?

### ISSUE THREAD

**Issue by tehmaestro** [CreateJS/SoundJS#186]
Hey. I think there's a problem with supporting no extension URLs.

The method `s.createInstance`, is only parsing the src with `var details = s._parsePath(src.src);` , which will fail for src's which have no extension. I think we need to use the `parseSrc` function, so that the src is validated, but there is a catch.

Calling `src = s._getSrcById(src);` only returns the src, without the extension, so `parseSrc` would always fail.

  I don't know if I make any sense, but is there an example for using extension-less URLs?
Thank you.

**Comment by wdamien** [comment]
Yes, this is a known issue. Its basically a duplicate of #120, since the fix will be the same for blob urls, and other urls without extensions.
